Makefile

```diff
@@ -10,15 +10,28 @@ include .version
 .PHONY: install-dependencies
 install-dependencies:
 	@rm -Rf bin && mkdir -p $(GOBIN)
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION)
-	go install github.com/goreleaser/goreleaser@$(GORELEASER_VERSION)
 	go install golang.org/x/tools/cmd/godoc@latest
 	go install golang.org/x/tools/cmd/stringer@latest
 
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION)
+	go install github.com/goreleaser/goreleaser@$(GORELEASER_VERSION)
+	go install github.com/google/go-licenses@latest
+
 .PHONY: build
 build:
 	@goreleaser build --rm-dist --snapshot --single-target --skip-validate
 
+.PHONY: gen-licenses
+gen-licenses:
+	./bin/go-licenses report . --template third_party_license.tpl > third_party_licenses.txt
```

Comment:
./bin/ is not needed and it may be wrong if GOPATH is different.

Reply:
right